src/utils.ts

```diff
@@ -361,16 +361,37 @@ function getCacheMode(flags: Flags): "offline" | boolean {
         "online"
       )}   Use a locally cached module if available and fresh.\n\t${c.bold(
         "offline"
-      )}  Use a locally cached module if available, even if stale.\n\t${c.bold(
+      )}   Use a locally cached module if available, even if stale.\n\t${c.bold(
         "no-cache"
-      )} Never use the local cache.`
+      )}   Never use the local cache.`
     );
 
   if (flags.cache === "offline") return "offline";
   if (flags.cache === "online") return true;
   return false;
 }
 
+const validPreloadModes = ["static", "dynamic", "no-preloads"];
+function getPreloadMode(flags: Flags): boolean | string {
+  if (!flags.preload) return "static";
+  if (!validPreloadModes.includes(flags.preload))
```

Comment:
Can we allow a "boolean" form ie `--preload` without a value? Perhaps we need to preparse this in the CLI handling?